src/client/public/oauth.html

```diff
@@ -1,2 +1,2 @@
-<script nomodule src="/scripts/oauth/oauth.js" crossorigin></script>
-<script type="module" src="/scripts/oauth/oauth.mjs" crossorigin></script>
+<script nomodule src="/bundled/scripts/oauth/oauth.js" crossorigin></script>
+<script type="module" src="/scripts/oauth/oauth.js" crossorigin></script>
```

Comment:
This is also nicer I think - DevTools doesn't recognize `.mjs` and won't link to the source properly.